release_images.yml

```diff
@@ -1,74 +1,89 @@
 ---
 release_images:
-# Check .release_images_template.yml for reference
+  # Check .release_images_template.yml for reference
   1:
     framework: "pytorch"
-    version: "1.12.1"
+    version: "1.11.0"
     arch_type: "x86"
-    customer_type: "ec2"
+    customer_type: "sagemaker"
     training:
-      device_types: ["cpu", "gpu" ]
-      python_versions: [ "py38" ]
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py38"]
       os_version: "ubuntu20.04"
-      cuda_version: "cu116"
-      example: False         # [Default: False] Set to True to denote that this image is an Example image
+      cuda_version: "cu113"
+      example: False # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
       # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      force_release: False # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
     inference:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py38"]
       os_version: "ubuntu20.04"
-      cuda_version: "cu116"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      cuda_version: "cu113"
+      example: False # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
       # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      force_release: False # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
   2:
```

Comment:
I don't see a Sagemaker training example image, please verify this exists

Reply:
thanks for catch that, i'll change to ec2